examples/plot_line_testing.py

```diff
@@ -0,0 +1,134 @@
+"""
+Test Line Plots with Matplotcheck
+=================================
+
+These are some examples of using the basic functionality of Matplotcheck
+to test line plots (including regression lines) in Python.
+"""
+
+################################################################################
+# Import Packages
+# ---------------
+# You will start by importing the required packages and plotting a line plot.
+
+import matplotlib.pyplot as plt
+import seaborn as sns
+import numpy as np
+from scipy import stats
+import pandas as pd
+import matplotcheck.base as pt
+
+################################################################################
+# Create Example Data
+# -------------------
+# Before you create a plot, you need to create some data. For plots with
+# regression lines, you will need data in which you are looking for trends
+# over time, such as maximum values from lidar-derived measurements.
+
+# Create dataframe of data points
+col1 = list(np.random.randint(25, size=15))
+col2 = list(np.random.randint(25, size=15))
+data = pd.DataFrame(list(zip(col1, col2)), columns=['Data1', 'Data2'])
```

Comment:
minor but let's keep all column names lower case so data1 and data2 vs Data1
i like to be consistent with case in code even if it's column names!